Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -539,8 +552,40 @@ static bool CompressCB(const std::string& text, float percent, void* ptr)
 
 void GameList::OnHeaderViewChanged()
 {
-  Settings::GetQSettings().setValue(QStringLiteral("tableheader/state"),
-                                    m_list->horizontalHeader()->saveState());
+  // So here's the deal: Qt's way of resizing stuff around stretched columns sucks ass
+  // That's why instead of using Stretch, we'll just make resizable columns take all the available
+  // space ourselves!
+
+  int available_width = width() - 5;
```

Comment:
Changing this to `int available_width = width() - style()->pixelMetric(QStyle::PM_ScrollBarExtent) - 1;` makes it not overflow into the vertical scrollbar.